unit-tests/live/d400/test-hdr.py

```diff
@@ -0,0 +1,638 @@
+        test.check(hdr_counter >= min_counter)
+    return True
+
+
+# CHECKING HDR MERGE AFTER HDR RESTART
+with test.closure("HDR Running - hdr merge after hdr restart", "[hdr][live][using_pipeline]"):
+    ctx = rs.context()
+    device = test.find_first_device_or_exit()
+    if ctx:
+        devices_list = ctx.query_devices()
+        device_count = devices_list.size()
+        if devices_list.size() != 0:
+            depth_sensor = device.first_depth_sensor()
+            if depth_sensor and depth_sensor.supports(rs.option.hdr_enabled):
+                # initializing the merging filter
+                merging_filter = rs.hdr_merge()
+
+                depth_sensor.set_option(rs.option.hdr_enabled, 1)
+                test.check(depth_sensor.get_option(rs.option.hdr_enabled) == 1)
+
+                cfg = rs.config()
+                cfg.enable_stream(rs.stream.depth)
+                pipe = rs.pipeline()
+                pipe.start(cfg)
+
+                frame_metadata_enabled = True
+                number_frames_streamed = 10
+
+                if not check_hdr_frame_counter(pipe, number_frames_streamed, merging_filter):
+                    frame_metadata_enabled = False
+
+                depth_sensor.set_option(rs.option.hdr_enabled, 0)
+                test.check(depth_sensor.get_option(rs.option.hdr_enabled) == 0)
+
+                if frame_metadata_enabled:
+                    if not check_hdr_frame_counter(pipe, number_frames_streamed, merging_filter):
+                        frame_metadata_enabled = False
+
+                depth_sensor.set_option(rs.option.hdr_enabled, 1)
+                test.check(depth_sensor.get_option(rs.option.hdr_enabled) == 1)
+
+                if frame_metadata_enabled:
+                    check_hdr_frame_counter(pipe, number_frames_streamed, merging_filter)
+
+                pipe.stop()
+                depth_sensor.set_option(rs.option.hdr_enabled, 0)  # disable hdr before next tests
+        else:
+            log.e("No device was found - skipping test")
+
+
+# CHECKING SEQUENCE ID WHILE STREAMING
+with test.closure("HDR Streaming - checking sequence id", "[hdr][live][using_pipeline]"):
+    ctx = rs.context()
+    device = test.find_first_device_or_exit()
+    if ctx:
+        devices_list = ctx.query_devices()
+        device_count = devices_list.size()
+        if devices_list.size() != 0:
+            depth_sensor = device.first_depth_sensor()
+            if depth_sensor and depth_sensor.supports(rs.option.hdr_enabled):
+                cfg = rs.config()
+                cfg.enable_stream(rs.stream.depth)
+                cfg.enable_stream(rs.stream.infrared, 1)
+                pipe = rs.pipeline()
+                pipe.start(cfg)
+
+                depth_sensor.set_option(rs.option.hdr_enabled, 1)
+                test.check(depth_sensor.get_option(rs.option.hdr_enabled) == 1)
+
+                sequence_id = -1
+                iterations_for_preparation = 30
```

Comment:
Can you explain the increase from 6 to 30?

Reply:
interesting...?